concordium-sdk/src/main/java/com/concordium/sdk/transactions/TransactionType.java

```diff
@@ -40,4 +40,13 @@ public enum TransactionType {
     TransactionType(byte type) {
         this.value = type;
     }
+
+    public static TransactionType parse(byte type) {
+        for (TransactionType transactionType : values()) {
+            if (transactionType.value == type) {
+                return transactionType;
+            }
+        }
```

Comment:
This makes me a bit sad :( 

Iterating through all the values to find the one we want when we in principle know the tags.

Reply:
me too  :(